pkg/build/webhook/generic/generic_test.go

```diff
@@ -28,7 +28,7 @@ func GivenRequest(method string) *http.Request {
 }
 
 func GivenRequestWithPayload(t *testing.T, filename string) *http.Request {
-	data, err := ioutil.ReadFile("fixtures/" + filename)
+	data, err := ioutil.ReadFile("../../../../test/fixtures/webhook/generic/" + filename)
```

Comment:
why move this? this seems considerable more fragile and more difficult to find in the source tree